### PR TITLE
Blog release 0.20.0

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@TomAugspurger started a more 'blog-post like' version of the whatsnew announcement for 0.20, with some additions from me. 
Idea is to have a shorter more digestable version of the most important things for people who would not read the full whatsnew notes.

Needs some further work, but feedback is certainly welcome 
@pandas-dev/pandas-core 

Rendered version: https://pandas-dev.github.io/pandas-website/pr-blog-release-0.20.0/blog/release-v0.20.0.html

**Comment by jreback** [comment]
will push some content a bit later

**Comment by jorisvandenbossche** [comment]
In the mean-time maybe a bit late, but would be nice to finish this off. 
@jreback did you had some updates?

**Comment by jreback** [comment]
let me have a look
got busy

**Comment by jreback** [comment]
updated the example for .agg

**Comment by jreback** [comment]
i need to edit

**Comment by jorisvandenbossche** [comment]
I did some updates on this. In the meantime it is already a bit late, but I think this is still useful content as a blog post. So would be nice to push this.

More comments?

@wesm ok with the wording about pandas 1.0 / 2.0 ?

**Comment by wesm** [comment]
@jorisvandenbossche LGTM

File: blog/release-v0.20.0.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
+The pandas 0.20 release
+=======================
+
```
Comment (jreback) [inline blog/release-v0.20.0.rst]: let's change to say 0.20.1 (or maybe 0.20.2 is fine)

**Comment by jorisvandenbossche** [comment]
Now 0.21 is there, lets merge this .. The content is still readable/valuable

**Pull request merged**